src/app/components/home/home.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Task } from './../../models/task.model';
+import { Component, OnInit } from '@angular/core';
+import { formatDate } from '@angular/common';
+import { TaskService } from 'src/app/services/task.service';
+import { UtilityService } from 'src/app/services/utility.service';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+
+})
+export class HomeComponent implements OnInit{
+
+    constructor(private _taskService: TaskService , private _utilityService: UtilityService){}
+    img_text = '';
```

Comment:
variables are usually declared before constructors

Reply:
I have replaced. But will discuss with you as I have found different opinion.